client/src/scenes/results/completed/index.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
```

Comment:
There's a component called `<CircularProgress>`, which is basically just a spinner. There's a lot of other components that use it to show a network request. Think you could add that here?

Reply:
Used a snackbar instead!